packages/web-media-box/src/pipelines/mse/consts/sourceBuffer.ts

```diff
@@ -0,0 +1,5 @@
+export const OperationType = {
+  APPEND: 'append',
+  ABORT: 'abort',
+  REMOVE: 'remove'
+};
```

Comment:
Is Enum a better fit here?

Reply:
Do we need this OperationType, don't really see usage...

Reply:
Yeah I was on the fence about this. I was originally planning on using operation type to make the `operation` function more generic, but I agree it's not necessary in this current iteration.